rosdep/ruby.yaml

```diff
@@ -77,7 +85,9 @@ ruby:
 ruby-sass:
   debian: [ruby-sass]
   fedora: [rubygem-sass]
+  gentoo: [dev-ruby/sass]
   ubuntu: [ruby-sass]
 ruby1.9.3:
+  gentoo: []
```

Comment:
do you know of any package using this ? I think this can be removes given that no released package on the buildfarm uses it